### PR TITLE
chore: Remove `--all` flag when starting `fpd` daemon

### PR DESCRIPTION
**Pull request by gitferry**
This PR removed the `--all` flag in `fpd start` so that all the registered fp instances will be started through `fpd start`. To start a specific instance, `--fp-btc-pk` flag should be specified.

This PR also changed the instructions accordingly.

### CODE REVIEW

File: docs/finality-provider.md [hunk 1]
```diff
@@ -150,13 +155,12 @@ can also be set in the configuration file.
...
-A finality provider named `my-finality-provider` can be created in the internal
-storage ([bolt db](https://github.com/etcd-io/bbolt))
-through the `fpcli create-finality-provider` command.
-This finality provider is associated with a BTC public key which
+We create a finality provider instance through the `fpcli create-finality-provider` or `fpcli cfp` command.
+The created instance is associated with a BTC public key which
 serves as its unique identifier and
 a Babylon account to which staking rewards will be directed.
-The key name must be the same as the key added in [step](#3-add-key-for-the-consumer-chain).
+Note that if the `--key-name` flag is not specified, the `Key` field of config specified in [step](#3-add-key-for-the-consumer-chain)
```
Comment (vitsalis) [inline docs/finality-provider.md:162]: Wrap to 80 chars?

File: docs/finality-provider.md [hunk 2]
```diff
@@ -135,6 +135,11 @@ This will start the RPC server at the address specified in the configuration und
 the `RpcListener` field, which has a default value of `127.0.0.1:15812`.
 You can also specify a custom address using the `--rpc-listener` flag.
 
+This will also start all the registered finality provider instances added in [step](#5-create-and-register-a-finality-provider).
+To start the daemon with a specific finality provider instance, use the `--fp-btc-pk`
+flag followed by the hex string of the BTC public key of the finality provider (`btc_pk_hex`)
+obtained in [step](#5-create-and-register-a-finality-provider).
```
Comment (vitsalis) [inline docs/finality-provider.md:141]: Wrap to 80 chars?

File: docs/finality-provider.md [hunk 3]
```diff
@@ -135,6 +135,11 @@ This will start the RPC server at the address specified in the configuration und
 the `RpcListener` field, which has a default value of `127.0.0.1:15812`.
 You can also specify a custom address using the `--rpc-listener` flag.
 
+This will also start all the registered finality provider instances added in [step](#5-create-and-register-a-finality-provider).
```
Comment (vitsalis) [inline docs/finality-provider.md:138]: Will it also start slashed ones?

**Review by vitsalis — APPROVED** [review]

File: docs/finality-provider.md (hunk 3 above)
Comment (gitferry) [reply docs/finality-provider.md]: Yep, added "except for slashed ones"

**Pull request merged**